File: submissions/igyuk-meg-a-compareto-t/6.cs
static void f(int input)
        {
            int tip = 0;
            if (input < 2000)
            {
                tip = ((input + 99) / 100) * 100 - input;
            } else if (input < 5000)
            {
                tip = ((input + 499) / 500) * 500 - input;
            } else
            {
                tip = (((int)(input * 1.1) + 499) / 500) * 500 - (int)(input);
            }

            if (tip == 0 || tip > 1500) {
                Console.WriteLine("PAID BY CARD");
            }
            else if(((double)tip /100) > Math.Floor((double)((double)tip / 100)))
            {
                Console.WriteLine("KEEP THE CHANGE YOU FILTY ANIMAL");
            } else
            {
                Console.WriteLine("HERE IS " + ((int)tip + (int)input));
            }
        }

File: submissions/hurf/6.js
function f(cost) {
    if (cost < 2000) {
        tip = Math.ceil(cost / 100) * 100;
    } else if (cost < 5000) {
        tip = Math.ceil(cost / 500) * 500;
    } else {
        tip = Math.round(Math.floor(cost * 1.1) / 500) * 500;
    }
    tip -= cost;
    //console.log(tip);
    if(tip == 0 || tip > 1500) {
        answer = "PAID BY CARD";
    } else if(tip%100 !== 0) {
        answer = "KEEP THE CHANGE YOU FILTY ANIMAL";
    } else {
        answer = "HERE IS " + (cost+tip);
    }
    console.log(answer);
};